content/docs/languages/cpp/quickstart.md

```diff
@@ -29,7 +29,7 @@ $ mkdir -p $MY_INSTALL_DIR
 Add the local `bin` folder to your path variable, for example:
 
 ```sh
-$ export PATH="$PATH:$MY_INSTALL_DIR/bin"
+$ export PATH="$MY_INSTALL_DIR/bin:$PATH"
```

Comment:
@chalin  I think this change is fair enough and it is indeed required to make sure e.g. user-installed cmake will be available as expected.